src/rez/package_test.py

```diff
@@ -103,8 +109,23 @@ def get_package(self):
             return self.package
 
         if self.use_current_env:
-            pass
+            # get package from current context, or return None
+            current_context = ResolvedContext.get_current()
+            if current_context is None:
+                return None
+
+            req = Requirement(self.package_request)
+            variant = current_context.get_resolved_package(req.name)
+            if variant is None:
+                return None
+
+            package = variant.parent
+
+            if not req.range.contains_version(package.version):
```

Comment:
No. I wanted to avoid the ambiguity, because VersionRange has methods that act both on Versions, and other VersionRanges. Of course you could have __contains__ be aware of both, but that starts feeling a little too implicit for my liking. I also didn't want to bloat this PR with unnecessary changes to version.py.

Reply:
I was just a general question :stuck_out_tongue: I was just surprised that `in` wasn't supported yet.